Apps/CommonData/src/Utils/DateFormatter.cs

```diff
@@ -37,7 +35,7 @@ public static class DateFormatter
     /// <returns>The date in YYYY-MM-DD format.</returns>
     public static string ToShortDate(DateOnly date)
     {
-        return date.ToString("d", CanadianCulture);
```

Comment:
We should probably remove CanadianCulture completely from this file as the parse could also have issues.

Reply:
Removed Canadian culture entirely from file and updated razor pages.